addons/mrp/models/mrp_routing.py

```diff
@@ -46,16 +46,16 @@ class MrpRoutingWorkcenter(models.Model):
         index=True, ondelete='cascade', required=True,
         help="The routing contains all the Work Centers used and for how long. This will create work orders afterwards "
         "which alters the execution of the manufacturing order.")
-    note = fields.Text('Description')
     company_id = fields.Many2one(
         'res.company', 'Company',
         readonly=True, related='routing_id.company_id', store=True)
-    worksheet = fields.Binary('PDF')
     worksheet_type = fields.Selection([
-        ('pdf', 'PDF'), ('google_slide', 'Google Slide')],
+        ('pdf', 'PDF'), ('google_slide', 'Google Slide'), ('text', 'Text')],
```

Comment:
How are you going to handle the migration script of databases that already have `note` and one of the other pre-existing `worksheet_type` used in them?

Reply:
I won't handle it, I can't force the change worksheet_type (no sense, client will be confused about the change), also delete the note is not a good idea (there is maybe a text version of explanation of the worksheet as a backup). Then If the client want the change the type into text, the note will still available. What do you think?

Reply:
Yeah, I think no matter what some users will be confused since if I read it correctly the text note won't be visible anymore unless the user changes the worksheet_type to text. I'm not sure how the migration notes to users work, but maybe it can be something worth mentioning?

Reply:
Yes of course, I will mention it in the commit message. Thanks!